README.md

```diff
@@ -28,18 +28,21 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
 tap "homebrew/cask"
-tap "telemachus/brew", "https://telemachus@bitbucket.org/telemachus/brew.git", pin: true
-tap "caskroom/fonts"
+tap "tap-from-non-standard-location", "https://user@bitbucket.org/user/homebrew-tap-repo.git"
```

Comment:
Sorry to be pedantic but can this still be in a `user/repo` format e.g. `user/tap-repo`

Reply:
Done!